2025R2/dpf-framework-25-r2/getting-started/dpf-docs-local-preview.md

```diff
@@ -0,0 +1,188 @@
+# DPF documentation: Generation and local preview
+
+This section details how to:
+
+- Generate the DPF Framework documentation including operator specifications, as Markdown files using command-line tools.
+- Build and serve a local HTML website to view the generated documentation using a static site generator such as Docfx.
+
+## Generating Documentation
+
+Follow the steps below to generate the DPF operator documentation in Markdown format.
+
+### 1. Set up the environment
+
+1. Clone the `pydpf-core` repository:
+
+   ```bash
+   git clone <repo-url>
+   ```
+
+2. Create and activate a virtual environment:
+
+   ```bash
+   python -m venv .venv
+   .venv\Scripts\activate.ps1
+   ```
+
+3. Set the required environment variables:
+
+   ```powershell
+   $env:ANSYS_DPF_ACCEPT_LA = "Y"
+   $env:ANSYSLMD_LICENSE_FILE = "1055@lyolmserv1.win.ansys.com"
+   ```
+
+### 2. Install the DPF server
+
+Navigate to the `ansys_dpf_server_win_v2025.1.pre0` folder and install the package in editable mode:
+
+```bash
+pip install -e .
```

Comment:
This is if you want to generate the doc for the standalone. If you want to generate the doc for a unified install then it will be different.